csfieldguide/templates/interactives/caesar-cipher.html

```diff
@@ -27,5 +27,5 @@ <h4>{% trans "Ciphertext" %}</h4>
 {% endblock css %}
 
 {% block js %}
-  <script type="text/javascript" src="{% static 'interactives/caesar-cipher/js/caesar.js' %}"></script>
+  <script type="text/javascript" src="{% static 'js/iframe-resizer-content-window.js' %}"></script>
```

Comment:
This should not required as it's already loaded in `csfieldguide/templates/interactives/base/iframe.html`.